Spec/Auth.swift

```diff
@@ -2441,6 +2441,59 @@ class Auth : QuickSpec {
                     }
                 }
 
+                it("example: if is initialised with custom ttl and a TokenParams object is passed with null, then the @ttl@ used for every subsequent authorization will be null") {
```

Comment:
This wording is just weird and does not reflect what http://docs.ably.io/client-lib-development-guide/features/#RSA10j says.

Why not just use the actual wording from the spec? "if a client is initialised with TokenParams#ttl configured with a custom value, and a TokenParams object is passed in as an argument to #authorize with a null value for ttl, then the ttl used for every subsequent authorization will be null"